### DISCUSSION TITLE
The sense of Mística Manager

### DISCUSSION THREAD

**Discussion by yceballost** [Telefonica/mistica-design#491] [Proposals]
# Context
El Mística Manager sale de la necesidad de cuando nació Mística de facilitar a los diseñadores de la compañía instalarse las librerías de Sketch, para ello se pensó que existiera un sitio donde con un click se te instalaran. Por ese momento no existía sitio de documentación por lo que no teníamos un lugar donde poner esos enlaces.

Hoy día tenemos el brand factory y Figma, y creo que el Manager a perdido su objetivo principal. 

Dicho esto, el manager es un buen sitio para poder centralizar futuras acciones, un software que está instalado en el equipo de los diseñadores que les de acceso a las diferentes utilidades de Mística.

Planteo varios debates:
- Tiene sentido el Mística Manager teniendo el Brand Factory?
- Tendría sentido evolucionar el Manager para que también tuvieses acceso a instalar desde ahí la app para ios / android y además abrir el Storybook de web? (Manager como distributiva al mundo Mística)
- Deberíamos tener en Brand Factory los enlaces a los UI Kits y otros materiales de Mística? (actualmente ya hay accesos a Storybook, etc.)
- Mantener el Manager como distributiva implicaría contenido duplicado con Brand Factory. (Asumible por mantenimiento yo creo)

**Comment by AnaMontes11** [comment]
Sería interesante que el Mistica Manager que ahora con Figma pierde en parte de su función principal, darle otras utilidades. 
Por ejemplo, estuve pensando si podría ser interesante que cuando tengamos una release pueda aparecer una notificación en el icono del manager y al acceder poder consultar la info de la release (link a GitHub) con su correspondiente mensaje de actualiza la librería (X), por ejemplo. 
Por otro lado, me parece una buena herramienta para poder tener centralizado todo lo relacionado con Mistica. Un Mistica Hub :)

**Comment by yceballost** [comment]
Created this issue to work on remove Manager
https://github.com/Telefonica/mistica-design/issues/1010
